keras_cv/backend/tf_ops.py

```diff
@@ -11,24 +11,39 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from keras_core.src.backend.tensorflow import *  # noqa: F403, F401
-from keras_core.src.backend.tensorflow import (  # noqa: F403, F401
-    convert_to_numpy,
-)
-from keras_core.src.backend.tensorflow.core import *  # noqa: F403, F401
-from keras_core.src.backend.tensorflow.math import *  # noqa: F403, F401
-from keras_core.src.backend.tensorflow.nn import *  # noqa: F403, F401
-from keras_core.src.backend.tensorflow.numpy import *  # noqa: F403, F401
+from keras_cv.backend import config
+
+if config.multi_backend():
```

Comment:
Do we need all the branching and try except?

Can we just do?

```
if detect_if_tensorflow_uses_keras_3():
    from keras.src.backend.tensorflow import *
    ...
else:
    from keras_core.src.backend.tensorflow import *
    ...
```

Reply:
done